cli/src/utils.rs

```diff
@@ -1,4 +1,4 @@
-use crate::CliResult;
+use crate::{CliResult, model::Model};
```

Comment:
Nitpicking: I really find these hard to parse, tbh. I don't mind flat {} but prefer when submodules get their own line. I'd like it if we can tweak your environment so it does not generate them...